JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj

```diff
@@ -12,9 +12,12 @@
     <PackageReference Include="AWSSDK.Core" Version="3.3.*" />
     <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.3.*" />
     <PackageReference Include="AWSSDK.SQS" Version="3.3.*" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
     <PackageReference Include="StructureMap" Version="4.7.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>

```

Comment:
Do we need this stuff? Is it new?

Reply:
It was generated by VS. I'd like to know.

Reply:
Because this is a test project and wouldn't be referenced by anyone else, `PrivateAssets` will have no effect. `IncludeAssets` defaults to `all`, and this looks like the explicit expanded version of that.

So this is equivalent to:
```xml
<PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" PrivateAssets="all" />
```
but it doesn't matter if you drop the `PrivateAssets="all"`